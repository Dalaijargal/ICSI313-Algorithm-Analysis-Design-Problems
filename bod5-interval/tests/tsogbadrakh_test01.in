10
0 3 3
4 7 2
1 7 5
8 10 2
9 10 1
11 14 3
13 15 2
16 19 3
18 20 2
21 22 1
