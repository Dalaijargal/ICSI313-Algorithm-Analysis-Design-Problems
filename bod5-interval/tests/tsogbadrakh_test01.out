15
1101111101